tools/system_libs.py

```diff
@@ -1002,18 +1119,18 @@ def get_files(self):
 
     return libc_files
 
-  def customize_build_cmd(self, cmd, filename):
-    if filename in self.non_lto_files:
+  def customize_build_flags(self, filename, flags):
+    if filename not in self.non_lto_files:
```

Comment:
Why is this flipped?

Reply:
I reverted all the changed to this function.